aws/ami/build_ami.sh

```diff
@@ -14,7 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PRODUCT=scylla
+PRODUCT=$(cat ../../SCYLLA-VERSION-GEN | grep PRODUCT= | sed -e 's/PRODUCT=//')
```

Comment:
why not run `SCYLLA-VERSION-GEN` ? 
and use `build/SCYLLA-PRODUCT-FILE` ?

also why not just, 
`source ../../SCYLLA-VERSION-GEN` 
?

Reply:
You are right, changed